content/src/content/jcr_root/apps/core/wcm/components/page/v1/page/head.html

```diff
@@ -15,7 +15,7 @@
   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
 <template data-sly-template.head="${ @ page }" data-sly-use.headlibRenderer="headlibs.html">
     <meta charset="UTF-8">
-    <title>${page.title}</title>
+    <title>${page.title}${page.brandSlug ? ' | ' : ''}${page.brandSlug}</title>
```

Comment:
or we adding this feature to Page v1 and v2, or only to v2?
If we add it to Page v1, we also need to update the dialog and add the `brandSlugTupple.js` to v1.

Reply:
backported feature in f39f298d8d1fae738c45fe04b61d3755d988b32f